examples/cpp_api/nullable_attribute.cc

```diff
@@ -178,11 +178,23 @@ void read_array() {
   std::cout << "a2: " << std::endl;
   for (i = 0; i < 4; ++i) {
     if (a2_validity_buf[i] > 0) {
-      std::cout << "{ ";
-      std::cout << std::to_string(a2_data[i * 2]);
-      std::cout << ", ";
-      std::cout << std::to_string(a2_data[i * 2 + 1]);
-      std::cout << " }";
```

Comment:
Were these lines causing problems? Ideally, we want the examples to be as trivial as possible and would prefer this over the new logic.

Reply:
No, these lines don't cause problems. I will revert it back to the old logic.